File: afisha-schedule/afisha-schedule-core/src/main/java/ru/kami/gis/afisha/schedule/core/repository/jdbc/api/CinemaDao.java
package ru.kami.gis.afisha.schedule.core.repository.jdbc.api;

import ru.kami.gis.afisha.schedule.api.repository.JdbcDaoInterface;
import ru.kami.gis.afisha.schedule.core.domain.CinemaEntity;

import java.util.List;

/**
 * @author Daniil.Makarov
 */
public interface CinemaDao extends JdbcDaoInterface {
    List<CinemaEntity> findAll();
}

File: afisha-schedule/afisha-schedule-core/src/main/java/ru/kami/gis/afisha/schedule/core/repository/jdbc/api/CinemaDao.kt
package ru.kami.gis.afisha.schedule.core.repository.jdbc.api

import ru.kami.gis.afisha.schedule.api.repository.JdbcDaoInterface
import ru.kami.gis.afisha.schedule.core.domain.CinemaEntity

/**
 * @author Daniil.Makarov
 */
interface CinemaDao : JdbcDaoInterface {
    fun findAll(): List<CinemaEntity?>
}